.github/workflows/tm2.yml

```diff
@@ -0,0 +1,56 @@
+name: tm2
+
+on:
+  pull_request:
+    paths:
+      - "go.sum"
+      - "tm2/Makefile"
+      - "tm2/**.go"
+      - ".github/workflows/tm2.yml"
+  push:
+    branches:
+      - master
```

Comment:
Does this also need to be `branches: [ $default-branch ]`?

Reply:
fixed 👍